packages/zapp/console/src/components/Executions/Tables/NodeExecutionsTable.tsx

```diff
@@ -26,17 +33,47 @@ const scrollbarPadding = scrollbarSize();
  * NodeExecutions are expandable and will potentially render a list of child
  * TaskExecutions
  */
-export const NodeExecutionsTable: React.FC<NodeExecutionsTableProps> = ({
+export const NodeExecutionsTable: FC<NodeExecutionsTableProps> = ({
```

Comment:
I prefer importing standard React hooks directly, but still keeping `React.` on type-like imports, so nobody is confused where it's coming from
```suggestion
export const NodeExecutionsTable: React.FC<NodeExecutionsTableProps> = ({
```

Reply:
fixed